### DISCUSSION TITLE
Better handling of logs

### DISCUSSION THREAD

**Discussion by swamper123** [FreeOpcUa/opcua-asyncio#444] [Ideas]
I had a chat with @AndreasHeine this week and @oroulet mentioned the problem once as well about the flood of logs that happen in asyncio.

It would be good, if we could choose in a nice way, what logs (and from which file) we want to see (and perhaps if we want to write them directly into a file).

**Comment by oroulet** [comment]
you can already do a lof of things using the python logger. set log level per file or directory in source : opcua.ua or opcua.server, etc...
